Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -164,6 +164,7 @@
 #import <pal/cocoa/RevealSoftLink.h>
 #import <pal/cocoa/VisionKitCoreSoftLink.h>
 #import <pal/cocoa/TranslationUIServicesSoftLink.h>
+#import <pal/cocoa/WritingToolsUISoftLink.h>
```

Comment:
Is this imoirt needed in this file?

Reply:
Yes (it's used in not-yet-upstreamed code in WebKitAdditions)

Reply:
And it will be more apparent in the update, since you already upstreamed that part.